psc_filing.yml

```diff
@@ -10,8 +10,8 @@ validation:
     'register-date-missing' : "Register entry date must be entered"
     'reference-psc-id-missing' : "Reference PSC ID must be entered"
     'reference-etag-missing' : "Reference ETag must be entered"
-    'super-secure-company' : "A PSC for this company has details protected. You can only file for this company on paper"
-    'company-type-not-allowed' : "This filing cannot be submitted for a "
+    'super-secure-company' : "A PSC for this company has details protected. You can only file for this PSC on paper"
+    'company-type-not-allowed' : "This filing cannot be submitted by this company type"
```

Comment:
Doesn't quite read naturally to me. Would it be clearer as "This filing cannot be submitted **for** this company type" ?

Reply:
yeah that does sound better - I've msgd Aron and testers.